server/cpho/jinja2/indicators/_indicator_form.jinja2

```diff
@@ -1,18 +1,26 @@
 {{ form.media }}
 
 {% macro form_field(field, rich=false) %}
-  <div class="row g-3 form-group mb-3">
-    <div class="col-lg-3">
-      <div class="row">{{ field.label_tag(attrs={'class': 'col col-form-label', 'aria-label': field.label }) }}</div>
-    </div>
-    <div class="col-lg-9">
-      {% if rich %}
-        {{ field|safe }}
-      {% else %}
-        {{ field }}
-      {% endif %}
+  {% if (field.name in form.hso_only_field_names and respects_rule('is_admin_or_hso') or (field.name not in form.hso_only_field_names)) %}
```

Comment:
Took me a while to understand this condition. Can we use intermediary variables to clean it up. Can also be simplified because admins can see all fields
```jinja2
{% set is_hso_only = field.name in form.hso_only_field_names %}
{% set is_admin = respects_rule('is_admin_or_hso') %}
{% if is_admin or not is_hso_only %}
 ....
{% endif %}
```